### ISSUE TITLE
Unable to close editable combobox for the 1st time & for second time on trying to close narrator focus moves to 1st list item.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#3523]
**Describe the bug**

**Steps to reproduce the bug**

Pre-requisites:
- On a touchscreen device, launch Narrator

1. Launch XAML Controls Gallery.
2. Activate Combo-box from All Controls Page
3. With item navigation (touch gesture swipe right) bring narrator focus to combo edit text box present under the header "An Editable Combo Box".
4. Double tap to activate it.
5. Now try to close it by tapping once outside and then double tap.
6. =>Pane is not closing.
7. Now traverse in the pane with item navigation and then select any value and double tap to activate.
8. Now Again activate the combo box traverse in the pane, single tap outside narrator announces close button, so double tap to close and observe.

**Expected behavior**

Visually impaired users should be able to close the combo box easily with touch gestures.

**Screenshots**

**Version Info**

Xaml Controls Gallery version 1.2.16.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/20679384)
There's a lengthy discussion in the internal issue about how to fix this.
